rosbag2_compression/include/rosbag2_compression/compression_options.hpp

```diff
@@ -51,6 +51,32 @@ ROSBAG2_COMPRESSION_PUBLIC CompressionMode compression_mode_from_string(
  */
 ROSBAG2_COMPRESSION_PUBLIC std::string compression_mode_to_string(CompressionMode compression_mode);
 
+enum class ROSBAG2_COMPRESSION_PUBLIC CompressionFormat: uint32_t
+{
+  NONE = 0,
+  ZSTD,
+  LAST_FORMAT = ZSTD
+};
```

Comment:
I don't think this is necessary; `CompressionFactory` should accept a string. This will play nicer if we ever introduce compression plugins and it would require the least amount of changes from what we have now.

Reply:
Makes sense. The reason I wanted an enum is to keep things consistent across the various rosbag2 packages.
If there's no other objections, I'll make the change.

Reply:
An enum worked for `CompressionMode` since we do not intend to implement that as a plugin.